src/App.vue

```diff
@@ -484,7 +484,13 @@ export default {
 
       this.setEntityName(business.legalName)
       this.setEntityType(business.legalType)
-      this.setEntityStatus(business.status)
+      if ('goodStanding' in business) {
```

Comment:
Out of curiousity, what is the value when they are not in compliance?

Reply:
goodStanding coming from api can either be true or false

Reply:
Why not simplify this check to "if true then ... else if false then ..." ?

Reply:
Trying to mitigate scenario where goodStanding not passed?

Reply:
If `goodStanding` is not passed then the value of `business.goodStanding` is undefined (not True or False), so you could simplify this.

I only picked on this because I did a double-take on `'goodstanding'`, thinking it needed a const or enum. Then I realized it was a property.

There's nothing wrong with your code. I'm closing this conversation.